src/styles/card-container.scss

```diff
@@ -0,0 +1,144 @@
+.card-container {
+  color: #060b35;
+  gap: 48px 46px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+
+  place-content: center;
+
+  padding: 50px 40px;
+}
+
+@media (min-width: 488px) {
+  .card-container {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .card-container {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .card-container {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.product-card {
+  display: flex;
+  box-sizing: border-box;
+  width: 200px;
+
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  flex-direction: column;
+  padding: 32px 16px 16px;
+
+  transition: all 0.3s;
+}
+
+.product-card:hover {
+  transform: scale(1.2);
+}
+
+.product-card__image {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+  align-self: center;
+  margin-inline: 3px;
+}
+
+.product-card__title {
+  text-align: left;
+  margin-bottom: 4px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+}
+
+.product-card:hover .product-card__title {
+  color: #34568b;
+}
+
+.product-card__code {
+  text-align: left;
+  font-size: 10px;
+  color: #616070;
+
+  font-weight: 400;
+  line-height: 14px;
+  margin-bottom: 16px;
+}
+
+.stars {
+  display: flex;
+  margin-bottom: 24px;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  justify-content: center;
+  align-items: center;
+  background-image: url(../images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  border-radius: 0.5px;
+  margin-right: 4px;
+}
+
+.stars__star--inactive {
+  background-image: url(../images/star.svg);
+}
+
+.reviews {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 16px;
+  margin-left: auto;
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  height: 18px;
+  margin-bottom: 16px;
+  line-height: 18px;
+}
+
+.price__text {
+  font-size: 12px;
+  font-weight: 400;
+  color: #616070;
+}
+
+.price__number {
+  font-size: 16px;
+  font-weight: 700;
+}
+
+.button-buy {
+  text-transform: uppercase;
+  text-align: center;
+  font-size: 14px;
+  line-height: 40px;
+  font-weight: 700;
+  color: #fff;
+
+  background-color: #00acdc;
+  border-radius: 5px;
+  height: 40px;
+  text-decoration: none;
+}
+
+.button-buy:hover {
+  color: #00acdc;
+  background-color: #fff;
+  border: 1px solid #00acdc;
+}
```

Comment:
The hover effect on the buy button is changing the color and the background color, which is fine. But it's also adding a border, which can cause the layout to shift slightly. Consider adding a transparent border to the button in its normal state to prevent this shift.